doc/ref_impl/cntt-ri/chapters/chapter03.md

```diff
@@ -4,16 +4,123 @@
+|n10, n20, n30, n40, n50, n60 | Y | Y | Y |
+|n25, n50, n75, n100, n125, n150 | N | Y | Y |
+|n50, n100, n150, n200, n250, n300 | N | Y | Y |
+|n100, n200, n300, n400, n500, n600 | N | Y | N |
+|CPU pinning support | N | Y | Y |
+|NUMA support | N | Y | Y |
+|IPSec Acceleration) | N | Y | N |
+|Crypto Acceleration | N | Y | N |
+|Transcoding Acceleration | N | N | Y |
+|Programmable Acceleration | N | N | Y |
+|Enhanced Cache Management | E | E | X |
+|Monitoring of L2-7 data | N | Y | N |
+|CPU overbooking | 1:4 | 1:1 | 1:1 |
+|vNIC QoS | N | Y | Y |
+|Huge page support | N | Y | Y |
+|Host CPU usage | Y | Y | Y |
+|Virtual compute CPU usage | Y | Y | Y |
+|Host CPU utilization | Y | Y | Y |
+|Virtual compute CPU utilization | Y | Y | Y |
+|External storage capacity | N | N | Y |
 
+> Open Point 1: Does ONAP have some relevant spec or VNF declaration schema so that CNTT can re-use/revise to cover what we need ? Or define a new one ?
+
+> Open Point 2: What principles should be followed if some the pre-define VNF profile items does not match what actual requires ? How to adjust, "ceiling", "floor", "customerize" ?
+
+> Open Point 3: Is it relevant/necessary to include NFVI SW/HW profile description file as following sections ?
 
 <a name="3.3"></a>
-## 3.3 Heading
+## 3.3 NFVI SW profile
+
+[RM1: 5.2 NFVI SW profile features and requirements](../../../ref_model/chapters/chapter05.html#5.2) defines the NFVI software layer. The profile depicts the feature status of the
+  - virtual Compute (**nfvi.com.cfg.xxx** in RM1 Table 5-7: Virtual Compute features and configuration for the 3 types of SW profiles and **nfvi.com.acc.cfg.xxx** in Table 5-8: Virtual Compute Acceleration features),
+  - storage (**nfvi.stg.cfg.xxx** in RM1: Table 5-9: Virtual Storage features and configuration for the 3 types of SW profiles and **nfvi.stg.acc.cfg.xxx** in Table 5-10: Virtual Storage Acceleration features)
+  - networking configuration(see **nfvi.net.cfg.xxx** in Table 5-11 Virtual Networking features and configuration for the 3 types of SW profiles and **nfvi.net.acc.cfg.xxx** in Table 5-12 Virtual Networking Acceleration features)
+
+This profile is the global settings for the whole NFVI, which means there should be only one entry per NFVI resource pool, i.e., Basic/Network/Compute
+
+| .conf | Basic | Network Intensive | Compute Intensive |
+|-------|----------------|----------------|----------------|
+| CPU allocation ratio  | 4:1 | 1:1  | 1:1 |
+| NUMA awareness | N | Y | Y |
+| CPU pinning capability | N | Y | Y |
+| Huge Pages | N | Y | Y |
+| Catalogue storage Types | Y  | Y  | Y |
+| Storage Block | Y | Y |Y  |
+| Storage Object | Y | Y | Y |
+| Storage with replication | N | Y | Y |
+| Storage with encryption | Y | Y | Y |
+| Storage IOPS oriented | N | Y | Y |
+| Storage capacity oriented | N | N | Y |
+| vNIC interface | virtio1.1 |  virtio1.1, i40evf (Intel driver for VF SR-IOV) |  virtio1.1, i40evf (Intel driver for VF SR-IOV) |
```

Comment:
Why explicitly mention virtio version 1.1? Does older version violates the requirements?